packages/bulkapi/src/controllers/BulkApi.ts

```diff
@@ -41,13 +43,31 @@ export class BulkApiImpl implements BulkApi {
             res.status(400)
             res.send({ issues: [validator.validationResult.issues.map(issue => issue.errorMsg())] })
         } else {
+            const issues: string[] = []
             for (const node of chunk.nodes) {
                 if (node.parent !== null && node.parent !== undefined) {
                     res.status(400)
-                    res.send({ issues: [`Node ${node} cannot be created as partition because it has a parent.`] })
-                    return
+                    issues.push(`Node ${node} cannot be created as partition because it has a parent.`)
+                }
+                for (const containment of node.containments) {
+                    if (containment.children.length !== 0) {
+                        issues.push(`Node ${node.id} cannot be created as a partition because it has children in containment ${containment.containment.key}`)
+                    }
+                }
+                if (node.annotations.length !== 0) {
+                    issues.push(`Node ${node.id} cannot be created as a partition because it has annotations`)
```

Comment:
I was not aware of this limitation on partitions

Reply:
The specification describes this as a restriction:

_We also MUST NOT mention any annotation/containment node ids in the partition nodes, as they cannot be part of the same request, and we don’t allow moving nodes in this operation. We MAY send properties and references{fn-org225}_